data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-creation-v1.0.xml

```diff
@@ -0,0 +1,211 @@
+                <constraints nullable="false"/>
+            </column>
+            <column name="group_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="leaf_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ref_transaction_id" type="VARCHAR(160)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ref_leaf_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="is_ref_input" type="BOOLEAN">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created" type="TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <addPrimaryKey columnNames="transaction_id, group_idx, leaf_idx"
+                       constraintName="utxo_transaction_sources_pkey"
+                       tableName="utxo_transaction_sources"/>
+        <addForeignKeyConstraint baseColumnNames="transaction_id,group_idx,leaf_idx"
+                                 baseTableName="utxo_transaction_sources"
+                                 constraintName="fk_utxo_transaction_sources"
+                                 referencedColumnNames="transaction_id,group_idx,leaf_idx"
+                                 referencedTableName="utxo_transaction_component"/>
+
+        <createTable tableName="utxo_transaction_output">
+            <column name="transaction_id" type="VARCHAR(160)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="group_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="leaf_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="type" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="token_type" type="VARCHAR(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="token_issuer_hash" type="VARCHAR(160)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="token_notary_x500_name" type="VARCHAR(1024)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="token_symbol" type="VARCHAR(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="token_tag" type="VARCHAR(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="token_owner_hash" type="VARCHAR(160)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="token_amount" type="NUMERIC">
+                <constraints nullable="true"/>
+            </column>
```

Comment:
We might want to think about breaking this out into a separate table once we have way of identifiying states that behave like tokens?

Reply:
created a tech debt ticket for this, I think what we have is ok for beta
https://r3-cev.atlassian.net/browse/CORE-7928